src/components/Movies/Movies.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import './Movies.css'
-import Card from '../Card/Card'
+import Card from '../Cards/Cards'
 
-const Movies = ({movies}) => {
+const Movies = ({movies, selectMovie}) => {
```

Comment:
We'll have to add in the selectMovies id so that this class component can hold our data fetched from the API. This is a solid foundation & set up to do so.